Program.cs

```diff
@@ -9,15 +9,47 @@ class Program
         int saldoLimite = 20000;
         bool MaximoCobro(TarjetaCredito credit) //Si el credito es menor al limite de saldo devuelve verdadero si no, devuelve falso
         {
-            if(credit.Saldo < saldoLimite)
+            if(credit.Saldo < Limite)
```

Comment:
1>C:\Users\chris\Cocoppel\Program.cs(12,31,12,37): error CS0103: The name 'Limite' does not exist in the current context